auth/auth_controller.go

```diff
@@ -54,19 +54,40 @@ func Login (w http.ResponseWriter, r *http.Request) {
 	fmt.Println(string(authJson))
 
 	// リクエスト送信
-	res, err := http.Post("http://localhost:3000/login.json", "application/json", bytes.NewBuffer(authJson))
+	response, err := http.Post("http://localhost:3000/login.json", "application/json", bytes.NewBuffer(authJson))
 	if err != nil {
 		return
 	}
-	defer res.Body.Close()
-	fmt.Println(res.Cookies())
+	defer response.Body.Close()
+
+	if (response.StatusCode != 200) {
+		log.Println("Unauthorized")
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusUnauthorized)
+		w.Write([]byte("401 - Unauthorized"))
+		return
+	}
 
 	// レスポンスの読み込み
-	body, err := ioutil.ReadAll(res.Body)
+	body, err := ioutil.ReadAll(response.Body)
```

Comment:
```suggestion
	body, err := io.ReadAll(response.Body)
```

ioutil は現在非推奨なので書き換えるといいと思います